packages/main/src/SliderBase.hbs

```diff
@@ -0,0 +1,31 @@
+<div
+	class="ui5-slider-root"
+	@mousedown="{{_onmousedown}}"
+	@touchstart="{{_ontouchstart}}"
+	@pointerdown="{{_onpointerdown}}"

```

Comment:
Well, there is not a specific reason, just that some of the other libraries support all of the event types and did it that way. I removed it. If ever in the future we must implement it it will be easy to add it anyway.